Swiftfin/Views/ItemEditorView/ItemSubtitles/ItemSubtitlesView.swift

```diff
@@ -0,0 +1,245 @@
+            }
+        } message: {
+            Text(L10n.deleteSelectedConfirmation)
+        }
+    }
+
+    // MARK: - Content Views
+
+    private var contentView: some View {
+        List {
+            ListTitleSection(
+                L10n.subtitles,
+                description: L10n.manageSubtitlesDescription
+            )
+
+            if !hasSubtitles {
+                Text(L10n.none)
+                    .frame(maxWidth: .infinity, maxHeight: .infinity, alignment: .center)
+                    .listRowSeparator(.hidden)
+                    .listRowInsets(.zero)
+            } else {
+                if viewModel.internalSubtitles.isNotEmpty {
+                    Section {
+                        DisclosureGroup(L10n.embedded) {
+                            ForEach(viewModel.internalSubtitles, id: \.index) { subtitle in
+                                SubtitleButton(subtitle) {
+                                    expandedSubtitle = subtitle
+                                }
+                                .environment(\.isEnabled, !isEditing)
+                            }
+                        }
+                    } footer: {
+                        Text(L10n.embeddedSubtitleFooter)
+                    }
+                }
+                if viewModel.externalSubtitles.isNotEmpty {
+                    Section {
+                        DisclosureGroup(L10n.external) {
+                            ForEach(viewModel.externalSubtitles, id: \.index) { subtitle in
+                                SubtitleButton(subtitle) {
+                                    if isEditing {
+                                        toggleSelection(for: subtitle)
+                                    } else {
+                                        expandedSubtitle = subtitle
+                                    }
+                                } onDelete: {
+                                    selectedSubtitles = [subtitle]
+                                    isPresentingDeleteConfirmation = true
+                                }
+                                .environment(\.isSelected, selectedSubtitles.contains(subtitle))
+                                .environment(\.isEditing, isEditing)
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    // MARK: - Subtitle Detail Sheet
+
+    @ViewBuilder
+    private var expandedSubtitleSheet: some View {
+        NavigationView {
+            if let mediaStream = expandedSubtitle {
+                MediaStreamInfoView(mediaStream: mediaStream)
+                    .navigationBarTitleDisplayMode(.inline)
+                    .navigationBarCloseButton {
+                        expandedSubtitle = nil
+                    }
+            }
+        }
+    }
+
+    // MARK: - Add/Remove Selected Subtitle
+
+    private func toggleSelection(for subtitle: MediaStream) {
```

Comment:
Use the `Set.toggle` function instead. There are other places throughout that it can be used instead.